periphery/src/managers/UniswapNFTManager.sol

```diff
@@ -36,10 +36,7 @@ contract UniswapNFTManager is IManager {
         // move position from NonfungiblePositionManager to Borrower
         if (liquidity < 0) {
             // safety checks since this contract will be approved to manager users' positions
-            {
-                require(FACTORY.isBorrower(msg.sender));
-                require(owner == NFT_MANAGER.ownerOf(tokenId));
-            }
+            require(FACTORY.isBorrower(msg.sender) && owner == NFT_MANAGER.ownerOf(tokenId));
```

Comment:
No reason for this to have been split into 2 requires instead of 1. Fixed.

Reply:
Well, there is a reason... it just isn't a very good one